cluster/swarm/cluster.go

```diff
@@ -186,6 +186,22 @@ func (c *Cluster) Image(IdOrName string) *cluster.Image {
 	return nil
 }
 
+func (c *Cluster) Pull(name string, callback func(string, string)) {
```

Comment:
Could you add a comment (either here or in the interface) and explain the callback (what are the arguments, it may be called multiple times, etc)?